lib/Assertion/ParameterProvider.php

```diff
@@ -14,6 +15,7 @@ public function provideFor(Variant $variant)
     {
         return (function (array $variantData) use ($variant) {
             return [
+                'subject' => $this->subjectData($variant->getSubject()),
                 'variant' => $variantData,
                 'baseline' => $variant->getBaseline() ? $this->buildVariantData($variant->getBaseline()) : $variantData,
```

Comment:
this is a bit of a smell - `subject` can have "properties" but `variant` is a list of results, maybe rename to `result`? this would free-up the `variant` key